### ISSUE TITLE
Cache installed languages.

### ISSUE THREAD

**Issue by simjanos-dev** [simjanos-dev/LinguaCafe#297]
It is really slow when a user wants to select a different language. Installed languages could be cached on startup, and be updated when installed languages are changed on the admin page. The admin pages would still use the non-cached results, which are slower but 100% accurate.

@sergiolaverde0 
Do you think this is a good solution? The only problem would be with this is that if the model folder would be deleted while the server is running, the cache would not be updated.

**Comment by sergiolaverde0** [comment]
Yes, that sounds right to me, at the beginning I proposed something similar and I still believe people deleting their files at runtime won't be a common occurrence, plus we have the fallback to the already existing system.

If this ever becomes an issue it is also possible that upon failed import we trigger a check for the installed languages, but I think it might be overkill.

**Comment by simjanos-dev** [comment]
Implemented.
